packages/application-shell/package.json

```diff
@@ -51,56 +51,49 @@
     "@commercetools-frontend/sentry": "2.1.1",
     "@commercetools-frontend/storage": "2.0.0",
     "@commercetools-frontend/url-utils": "2.0.0",
-    "@flopflip/launchdarkly-adapter": "2.4.4",
-    "@flopflip/memory-adapter": "1.1.0",
-    "@flopflip/react-broadcast": "7.2.3",
     "apollo-cache-inmemory": "1.3.10",
-    "apollo-client": "2.4.6",
     "apollo-link": "1.2.3",
     "apollo-link-error": "1.1.1",
     "apollo-link-http": "1.5.5",
     "apollo-link-logger": "1.2.3",
     "apollo-link-retry": "2.2.5",
-    "classnames": "2.2.6",
-    "common-tags": "1.8.0",
-    "debounce-async": "0.0.2",
-    "downshift": "3.1.5",
-    "fast-equals": "1.6.1",
-    "formik": "1.3.2",
```

Comment:
The above are all shared and used across both so both lib/app have a requirement towards them.